pydicom/pixel_data_handlers/pillow_handler.py

```diff
@@ -133,21 +133,11 @@ def get_pixeldata(ds):
                .format(transfer_syntax.name))
         raise NotImplementedError(msg)
 
-    if transfer_syntax not in PillowSupportedTransferSyntaxes:
-        msg = ("this transfer syntax {0}, can not be read because "
-               "Pillow does not support this syntax"
-               .format(transfer_syntax.name))
-        raise NotImplementedError(msg)
-
-    if transfer_syntax in PillowJPEGTransferSyntaxes:
-        logger.debug("This is a JPEG lossy format")
-        if ds.BitsAllocated > 8:
-            raise NotImplementedError("JPEG Lossy only supported if "
-                                      "Bits Allocated = 8")
-    elif transfer_syntax in PillowJPEG2000TransferSyntaxes:
-        logger.debug("This is a JPEG 2000 format")
-    else:
-        logger.debug("This is a another pillow supported format")
+    if transfer_syntax == pydicom.uid.JPEGExtended and ds.BitsAllocated > 8:
+        raise NotImplementedError(
+            "{} - {} only supported if Bits Allocated = 8"
+            .format(pydicom.uid.JPEGExtended, pydicom.uid.JPEGExtended.name)
```

Comment:
can we add "by Pillow" to the "only supported line?".

Reply:
Also, maybe "!= 8" is stronger than ">8"? (should never happen, but lots of non-standard things do).